launch_chffrplus.sh

```diff
@@ -77,9 +77,7 @@ function launch {
   export PYTHONPATH="$PWD:$PWD/pyextra"
 
   # hardware specific init
-  if [ -f /TICI ]; then
-    tici_init
-  fi
+  tici_init
```

Comment:
`agnos_init` ?

Reply:
line 11

function tici_init {

-> function agnos_init {

Reply:
line 11

function tici_init {

-> function agnos_init {

Reply:
no reason this shouldn't work on PC